website/project/model.py

```diff
@@ -2158,8 +2159,13 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             foreign_user=foreign_user,
             params=params,
         )
+
         if log_date:
             log.date = log_date
+
+        self.date_modified = log.date.replace(tzinfo=None)
```

Comment:
That isn't really affecting the node in a way that should change its modified date, so I guess it makes the most sense to do that update here.

However, considering this functions `save` kwarg defaults to `True`, should it be calling `.save()` outside of the `if save:` block?